Core/Helper.php

```diff
@@ -46,6 +48,31 @@ public static function createDeviceFingerprint($sMaid, $sSessionId = null)
         return $sMaid . '_' . $sSessionId;
     }
 
+    /**
+     * Recursively calls itself until a transaction without a parent transaction ID is found.
+     * This is the root transaction and its transaction ID is returned.
+     *
+     * @param string $sTransactionId
+     *
+     * @return string root transaction ID
+     */
+    public static function getRootTransactionId($sTransactionId)
```

Comment:
i see this method more in the transaction handler as a private one, where do you plan on using it as it is in a helper?

Reply:
I agree, will move it there.